src/app/pages/send-tokens/send-tokens.tsx

```diff
@@ -136,6 +141,7 @@ function SendTokensFormBase() {
               <SendFormInner
                 assetError={assetError}
                 feeEstimations={feeEstimations.estimates}
+                onAssetIdSelected={onAssetIdSelected}
```

Comment:
```suggestion
                onAssetIdSelected={assetId => setSelectedAssetId(assetId)}
```

this reads much better than repeating the event name

Reply:
Good point, thx, will change.